docs/articles_en/openvino_workflow/running_inference_with_openvino/string_tensors.rst

```diff
@@ -0,0 +1,217 @@
+.. {#openvino_docs_OV_UG_string_tensors}
+
+String Tensors
+==============
+
+
+.. meta::
+   :description: Learn how to pass and retrieve text to and from OpenVINO model.
+
+OpenVINO tensor can hold as its elements not only numerical data like floating point or integer numbers,
+but also can contain textual information that is represented as one or multiple strings.
+Such a tensor is called a string tensor.
```

Comment:
```suggestion
Such a tensor is called as a string tensor.
```

Reply:
https://ell.stackexchange.com/questions/90244/which-is-the-correct-sentence-is-is-called-called-as